cmake/SetupCompilerFlags.cmake

```diff
@@ -17,3 +17,7 @@ if (ENABLE_COVERAGE)
   set(CMAKE_CXX_FLAGS "-coverage ${CMAKE_CXX_FLAGS}")
   set(CMAKE_EXE_LINKER_FLAGS "-coverage ${CMAKE_EXE_LINKER_FLAGS}")
 endif ()
+
+if (ENABLE_HIP)
+	set(HIP_HIPCC_FLAGS "${HIP_HIPCC_FLAGS} -Wno-inconsistent-missing-override")
```

Comment:
I take it that running the compiler without this flag generates a lot of missing override warnings?  I'm curious, should we consider fixing these?

Reply:
Good question. I'll defer that one to @noelchalmers

Reply:
I'm fairly sure this is a bug on AMD's end. The use of ```override``` seems correct in umpire. I'll look into what's causing this with the hipcc compiler team.

Reply:
Awesome, we'll keep this flag for now then.

Reply:
May we add a comment here or write a Jira ticket so that we don't forget about it?

Reply:
I'll file a Jira ticket today for this.

Reply:
I think Marty meant a JIRA ticket for us 😄, but if you can get this sorted on your end that would be excellent!

Reply:
Ah, I didn't catch that :smiley: . Yeah since I think this issue is in our compiler, I'll make a ticket here so that someone on our compiler team can confirm/deny (and fix?) this.